### PR TITLE
Update GitHub fork ribbon to v0.2.0.

### PR DESCRIPTION
**Pull request by XhmikosR**
Also, add it in removal page.

### CODE REVIEW

**Comment by XhmikosR** [comment]
I don't get why tests are failing. :/

**Comment by lgarron** [comment]
Usually a flakiness problem; we can ignore it for this.

File: frontend/index.html [hunk 1]
```diff
@@ -21,6 +21,7 @@
   </script>
 </head>
 <body class="theme-green">
+<a class="github-fork-ribbon" href="https://github.com/chromium/hstspreload.org" data-ribbon="On GitHub" title="On GitHub">On GitHub</a>
```
Comment (lgarron) [inline frontend/index.html]: For my own future reference: looks like getting a text attribute from CSS is well-supported:
https://developer.mozilla.org/en-US/docs/Web/CSS/attr

**Comment by lgarron** [comment]
Thanks!

**Pull request merged**